posts/2021-07-22-cloud-hosted-guides-post.adoc

```diff
@@ -1,33 +1,27 @@
 open-graph-image: https://openliberty.io/img/twitter_card.jpg
 ---
-= We now have fully interactive guides running in the cloud!
+= Interactive developer guides running in the cloud
 Jamie Coleman <https://github.com/jamiecoleman92>
 :imagesdir: /
 :url-prefix:
 :url-about: /
 //Blank line here is necessary before starting the body of the post.
 
-Our dedicated team of engineers have been hard at work creating new content for developers to enjoy. We now have 57 different guides on a range of topics through development, built, test and deploy so you as a developer can get started quickly and have a great experience with our tech. We now offer the possibility to run our guides in a hosted environment wihtout having to worry about local setup and prerequisits. We call these "cloud-hosted guides"and you can access them by going to one of our cloud enabled guides on link:{url-prefix}/guides/[Open Liberty] and clicking the “Run in cloud” button in the topright as shown in the image below:
+== Get hands-on with open, cloud-native technologies without the need for any local pre-requisites
 
-image::/img/blog/OL_Cloud-hosted-1.png[Image 1,width=70%,align="center"]
-
-== What is the Skills Network and why are we using it with our guides?
-
-The Skills Network is a learning environment that runs through a user’s browser. It runs in containers that are deployed to OpenShift in the IBM Cloud. This allows us not only to teach users how to build great cloud-native applications but also demonstrates that in practice with how the environment is deployed. Having control over every user’s environment means that we can create a place where the user can just get on with learning new skills rather than having to set things up locally. 
+Our team of engineers have been hard at work creating new content for developers to enjoy. We now have 57 different guides On a range of topics and technologies throughout development, build, testing and deployment, designed to help developers get started quickly with Open Liberty and have a great experience with this technology. Excitingly, we now offer the possibility to run our guides in a hosted environment wihtout having to worry about local setup and prerequisits. We’ve named these our ‘cloud-hosted guides’ and you can access them by going to one of our cloud enabled guides on link:{url-prefix}/guides/[Open Liberty] and clicking the “Run in cloud” button in the topright as shown in the image below:
 
-image::/img/blog/OL_Cloud-hosted-2.png[Image 2,width=70%,align="center"]
-
-The user has access to all the tools required to build cloud-native applications such as Docker, Kubernetes, OpenShift, Maven, Gradle, OpenJ9 JVM and the Open Source Theia IDE which is based on VS Code and also runs in a container. 
+image::/img/blog/OL_Cloud-hosted-1.png[Image 1,width=70%,align="center"]
 
 == What guides are available?
 
```

Comment:
We shouldn't need to mention the silent launch.  Let us state simply that 22 guides are now available as interactive guides you can access through a cloud-hosted environment.

Reply:
Done